pkg/api/handlers/libpod/kube.go

```diff
@@ -100,6 +101,7 @@ func KubePlay(w http.ResponseWriter, r *http.Request) {
 		PublishPorts:     query.PublishPorts,
 		Wait:             query.Wait,
 		ServiceContainer: query.ServiceContainer,
+		PublishAll:       query.PublishAll,
```

Comment:
For this to work remotely, we also need to touch `pkg/bindings`.  Those are the go-bindings `podman-remote` uses.  In case the client does not set the publish-all parameter, we should query contianers.conf here and use the configured defaults on the server side.

Reply:
Since it is mostly generated code I might need some guidance.

The `PlayOptions` in `pkg/binding/kube/types.go` needs to be modified right?

After that we need to check if the flag is checked in the `PlayWithBody` func in the `pkg/binding/kube/kube.go` if I'm not mistaken.

Where and how should I run the generate command?

Currently I'm not sure how and where should I query the `containers.conf` may I require some aditional information about where should we do that exactly?